src/pages/Home/ItemCard.jsx

```diff
@@ -1,17 +1,34 @@
-import React from 'react'
+import React,{useState,useEffect} from 'react';
+import { useQuery } from "@tanstack/react-query";
+import { getDashboard } from "../../api/dashboard.js";
 
 const ItemCard = (props) => {
-    
+
+  const { data:CardData, isloading } = useQuery({
+    queryKey: ["CardData"],
+    queryFn: () => getDashboard(),
+    onError: (error) => {
```

Comment:
the onError and OnSuccess in only available in useMutation hook not in useQuery, remove them, intead you can use `isError` prop returned by the hook to handle errors